### ISSUE TITLE
Unable to download webdriver for Brave (`no such driver version 115.56.14 for <platform>`)

### ISSUE THREAD

**Issue by ndepaola** [SergeyPirogov/webdriver_manager#579]
## System
* `webdriver_manager==4.0.0` (latest)
* `Brave Browser==115.0.5790.114` (latest)
* I can reliably reproduce this issue on all of the following operating systems:
    * macOS 12.6.3
    * Windows 10 22H2
    * Ubuntu 23.04

---

## Description
Howdy, I'm trying to use `webdriver_manager` to manage my Brave webdrivers (as my project supports the three major Chromium browsers), and `webdriver_manager` works correctly for Chrome and Edge but not Brave. I'm facing the following error:

`No such driver version 115.1.56.14 for <platform>`

My Brave configuration code (which matches my Chrome and Edge configuration code except for hardcoding paths to the Brave executable for each platform):

```py
def get_brave_driver(headless: bool = False, binary_location: Optional[str] = None) -> Chrome:
    options = Options()
    options.add_argument("--log-level=3")
    options.add_argument("--disable-dev-shm-usage")
    if headless:
        options.add_argument("--headless")
    options.add_experimental_option("excludeSwitches", ["enable-logging"])
    options.add_experimental_option("detach", True)

    # the binary location for brave must be manually specified
    # the below code block is partially borrowed from webdriver-manager tests:
    # https://github.com/SergeyPirogov/webdriver_manager/blob/master/tests/test_brave_driver.py
    if binary_location is not None:
        options.binary_location = binary_location
    else:
        default_binary_location = {
            OSType.LINUX: "/usr/bin/brave-browser",
            OSType.MAC: "/Applications/Brave Browser.app/Contents/MacOS/Brave Browser",
            OSType.WIN: "C:\\Program Files\\BraveSoftware\\Brave-Browser\\Application\\brave.exe",
        }[OperationSystemManager.get_os_name()]
        options.binary_location = default_binary_location

    driver = Chrome(service=Service(ChromeDriverManager(chrome_type=ChromeType.BRAVE).install()), options=options)  # type: ignore
    driver.set_network_conditions(offline=False, latency=5, throughput=5 * 125000)
    return driver
```

Let me know if you need any more detail from me. Thanks in advance!

**Comment by 54Ezreal** [comment]
I also have this problem

**Comment by ucchiee** [comment]
same as above.

**Comment by lee-eve** [comment]
> 
Starting with M115 the ChromeDriver release process is integrated with that of Chrome. 
Details can be found in the link below :
https://sites.google.com/chromium.org/driver/downloads/version-selection?authuser=0

**Comment by mantou568** [comment]
I also have this problem.How should we solve this problem? I found that the latest free 114 version in the URL does not have the 115 version

**Comment by garysassano** [comment]
Using the code in the README file:
```
# selenium 4
from selenium import webdriver
from selenium.webdriver.chrome.service import Service as ChromiumService
from webdriver_manager.chrome import ChromeDriverManager
from webdriver_manager.core.utils import ChromeType

driver = webdriver.Chrome(service=ChromiumService(ChromeDriverManager(chrome_type=ChromeType.CHROMIUM).install()))
```
I get the following error:
```sh
ImportError: cannot import name 'ChromeType' from 'webdriver_manager.core.utils'
```

**Comment by Robotz213** [comment]
(Caso a tradução esteja ruim, me avise)Try using Chromium 114, it worked really well for me and change this line

from webdriver_manager.core.utils import ChromeType

to

from webdriver_manager.core.os_manager import ChromeType


and reinstall webdriver_manager for loading updates

**Comment by devpawann** [comment]
> Using the code in the README file:
> 
> ```
> # selenium 4
> from selenium import webdriver
> from selenium.webdriver.chrome.service import Service as ChromiumService
> from webdriver_manager.chrome import ChromeDriverManager
> from webdriver_manager.core.utils import ChromeType
> 
> driver = webdriver.Chrome(service=ChromiumService(ChromeDriverManager(chrome_type=ChromeType.CHROMIUM).install()))
> ```
> 
> ![](https://storage.googleapis.com/pieces-web-extensions-cdn/pieces.png)
> Copy and Save
> ![](https://storage.googleapis.com/pieces-web-extensions-cdn/link.png)
> Share
> I get the following error:
> 
> ImportError: cannot import name 'ChromeType' from 'webdriver_manager.core.utils'

Change the import
`from webdriver_manager.core.os_manager import ChromeType`

**Comment by ndepaola** [comment]
@SergeyPirogov could you explain how this is fixed please? I can't see any commits in master that fix Brave webdriver support since 4.0.0
